docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,144 @@
+
+    ```
+    python -m pip install mbed-tools
+    ```
+
+- To install a specific version:
+
+    ```
+    python -m pip install mbed-tools==<version number in major.minor.patch format>
+    ```
+
+- To install a pre-release or development version:
+
+    ```
+    python -m pip install mbed-tools --pre
+    ```
+
+## Upgrade
+
+Use pip to upgrade:
+
+```
+python -m pip install mbed-tools --upgrade
+```
+
+# Use
+
+To get help for a specific command, use `mbed-tools <command> --help`. For example, for helping with listing connected devices (the `devices` command):
+
+```
+mbed-tools devices --help
+```
+
+## Create a new project
+
+To create a new Mbed OS project in a specified path:
+
+- To create the project and download a new copy of Mbed OS (latest official release):
+
+    ```
+    mbed-tools init <PATH>
+    ```
+
+- To create a project without downloading a copy of Mbed OS (reuse an existing copy):
+
+    ```
+    mbed-tools init -c <PATH>
+    ```
+
+    <!--so how do I tell it where the existing copy is? In Mbed Studio you have to explicitly "tell it" where the local copy is-->
+
+## List connected devices
+
+To list devices connected over USB:
+
+```
+mbed-tools devices
+```
+
+<!--why? what do I do with this info? is this for the config?-->
+
+## Set environment variables
+
+<!--like what? when? why? should this be with installation instructions?-->
+
+To override default environment variables:
+
+```
+mbed-tools env
+```
+
+## Configure Mbed OS
+
+<!--I don't understand what this actually does. Does it generate a config file? why do I need to do this? what does "for use with" mean - do I use Mbed Tools with the target and toolchain, or am I configuring Mbed OS itself? Do I always have to do this before I build, or can I do it at the same time as the build, or can I just skip it sometimes?-->
+
+To prepare the Mbed configuration information for use with a specific target and toolchain:
+
+```
+mbed-tools configure -m <target> -t <toolchain>
+```
+
+Currently the supported targets are `K64F`, `DISCO_L475VG_IOT01A`, `NRF52840_DK`
+
+<!--just for configuring, or can you only build with those?-->
+
+## Build using CMake
+
+After preparing the Mbed configuration information, you can use CMake to build your application:<!--can, but don't have to? what are the other options?-->
+
+```console
+cmake -S . -B cmake_build -GNinja -DCMAKE_BUILD_TYPE=<profile>
+cmake --build cmake_build
+```
+<!--what are all the other parameters?-->
+<!--what does the first command do, and what does the second one do?-->
+- We recommend using a dedicated **cmake_build** folder to keep things organised.<!--so do I replace cmake_build with a new value?-->
+- The `profile` value can be `release`, `debug` or `develop`.<!--same as normal, or something specific to CMake? https://os.mbed.com/docs/mbed-os/v6.2/program-setup/build-profiles-and-rules.html-->
+
+<!--what does this generate and where? is this where we add instructions for flashing the board?-->
```

Comment:
It generates the binary and hex files in the `cmake_build` folder. "Flashing" (i.e invoking `cp` to copy the binary to the USB disk presented by DAPlink) isn't implemented in the new tools yet, but it's on our backlog.

Reply:
So flashing is drag&drop?

Reply:
At the moment flashing is not supported by the tool and is left to the user. They can use drag&drop.